src/integration/java/uk/nhs/prm/repo/ehrtransferservice/parsers/ParserBrokerIntegrationTest.java

```diff
@@ -121,36 +118,52 @@ void shouldPublishCopcMessageToLargeMessageFragmentTopic() throws IOException {
     }
 
     @Test
-    void shouldPublishSmallMessageToSmallEhrObservabilityQueue() throws IOException {
-        var smallEhrMessageBody = dataLoader.getDataAsString("RCMR_IN030000UK06");
-
-        var inboundQueueFromMhs = new SimpleAmqpQueue(inboundQueue);
-        inboundQueueFromMhs.sendMessage(smallEhrMessageBody);
-
-        var smallEhrObservabilityQueueUrl = sqs.getQueueUrl(smallEhrObservabilityQueueName).getQueueUrl();
+    void shouldPublishEhrCoreToSmallEhrObservabilityQueue() throws IOException {
+        // given
+        final RepoIncomingEvent repoIncomingEvent = createDefaultRepoIncomingEvent(EHR_CORE_INBOUND_CONVERSATION_ID);
+        final String ehrCoreMessageBody = dataLoader.getDataAsString("RCMR_IN030000UK06");
+        final SimpleAmqpQueue inboundQueueFromMhs = new SimpleAmqpQueue(inboundQueue);
+        final String smallEhrObservabilityQueueUrl = sqs.getQueueUrl(smallEhrObservabilityQueueName).getQueueUrl();
+
+        // when
+        transferService.createConversation(repoIncomingEvent);
+        transferService.updateConversationTransferStatus(
+            EHR_CORE_INBOUND_CONVERSATION_ID,
+            INBOUND_REQUEST_SENT
+        );
+
+        inboundQueueFromMhs.sendMessage(ehrCoreMessageBody);
 
         await().atMost(10, TimeUnit.SECONDS).untilAsserted(() -> {
             var receivedMessageHolder = checkMessageInRelatedQueue(smallEhrObservabilityQueueUrl);
-            Assertions.assertTrue(receivedMessageHolder.get(0).getBody().contains(smallEhrMessageBody));
+            Assertions.assertTrue(receivedMessageHolder.get(0).getBody().contains(ehrCoreMessageBody));
             Assertions.assertTrue(receivedMessageHolder.get(0).getMessageAttributes().containsKey("traceId"));
             Assertions.assertTrue(receivedMessageHolder.get(0).getMessageAttributes().containsKey("conversationId"));
         });
     }
 
     @Test
     void shouldPassCorrelationIdToBeSetAsTraceId() throws IOException {
-        var correlationId = UUID.randomUUID().toString();
-        var smallEhrMessageBody = dataLoader.getDataAsString("RCMR_IN030000UK06");
-
-        var inboundQueueFromMhs = new SimpleAmqpQueue(inboundQueue);
-        inboundQueueFromMhs.sendMessage(smallEhrMessageBody, correlationId);
-
-        var smallEhrObservabilityQueueUrl = sqs.getQueueUrl(smallEhrObservabilityQueueName).getQueueUrl();
+        // given
+        final RepoIncomingEvent repoIncomingEvent = createDefaultRepoIncomingEvent(EHR_CORE_INBOUND_CONVERSATION_ID);
+        final String ehrCoreMessageBody = dataLoader.getDataAsString("RCMR_IN030000UK06");
+        final SimpleAmqpQueue inboundQueueFromMhs = new SimpleAmqpQueue(inboundQueue);
+        final String smallEhrObservabilityQueueUrl = sqs.getQueueUrl(smallEhrObservabilityQueueName).getQueueUrl();
+        final String correlationId = UUID.randomUUID().toString();
```

Comment:
We should try to move away from the old team's terminology of calling a conversation id a correlation id.
```suggestion
        final String conversationId = UUID.randomUUID().toString();
```

Reply:
Let's keep this in mind! 😄